cellprofiler/modules/watershed.py

```diff
@@ -21,7 +18,52 @@
 YES          YES          YES
 ============ ============ ===============
 
-.. _scikit-image documentation: http://scikit-image.org/docs/dev/auto_examples/segmentation/plot_watershed.html
+Select a method of inputs for the watershed algorithm:
```

Comment:
This appears to be duplicated form L105? Because the .format block is only on the latter this won't properly fill in the names of settings.

Reply:
One was supposed to show the individual section help and the other for the module help.